.github/workflows/lint-pr-title-preview.yml

```diff
@@ -14,6 +14,8 @@ jobs:
       - uses: actions/setup-node@v1
         with:
           node-version: 12
+      - run: yarn install
+      - run: yarn build
```

Comment:
In the current branch we have to build the action as it's referenced with `dist/index.js`.